ios/RNCWKWebView.m

```diff
@@ -106,7 +106,63 @@ - (void)didMoveToWindow
     wkWebViewConfig.mediaPlaybackRequiresUserAction = _mediaPlaybackRequiresUserAction;
 #endif
 
-    _webView = [[WKWebView alloc] initWithFrame:self.bounds configuration: wkWebViewConfig];
+    if(_sharedCookiesEnabled) {
+        // More info to sending cookies with wkwebview: https://stackoverflow.com/questions/26573137/can-i-set-the-cookies-to-be-used-by-a-wkwebview/26577303#26577303
+        if (@available(iOS 11.0, *)) {
+          NSArray<NSHTTPCookie*>* cookies = [[NSHTTPCookieStorage sharedHTTPCookieStorage] cookies];
+          for(int i = 0; i < (int)[cookies count]; ++i)
+          {
+              NSHTTPCookie* currentCookie = cookies[i];
+              [wkWebViewConfig.websiteDataStore.httpCookieStore setCookie: currentCookie completionHandler: nil];
+          }
+
+          _webView = [[WKWebView alloc] initWithFrame:self.bounds configuration: wkWebViewConfig];
+
+          // Hotfix: set cookies twice, otherwise they are not sent with first request
```

Comment:
Why does this need to be set twice?

It does look like the `setCookie:completionHandler:` is an async method, so is it because there's a race condition here?

Reply:
Yeah you are right. I found another solution here (https://forums.developer.apple.com/thread/97194)
This line of code should do the trick: 
`wkWebViewConfig.websiteDataStore = [WKWebsiteDataStore nonPersistentDataStore];`
Ititialze the websiteDataStore worked for me.

Reply:
I updated the PR, do you need further information?
Will you merge the PR into master?